test/test_utils.jl

```diff
@@ -11,6 +11,13 @@ using FeatureTransforms.TestUtils
 
         M = reshape(1:9, 3, 3)
         @test FeatureTransforms.apply(M, t) == ones(3, 3)
+
+        # TODO: remove when refactoring tests
```

Comment:
I've already started this https://github.com/invenia/FeatureTransforms.jl/pull/84